apache/assets/recommended_monitors/high_keep_alive_and_cpu.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "[Apache] resource utilization",
+	"type": "query alert",
+	"query": "avg(last_4h):anomalies(+ avg:apache.performance.cpu_load{*} by {host}, 'agile', 2, direction='above', alert_window='last_15m', interval=60, count_default_zero='true', seasonality='hourly') >= 1",
+	"message": "High number of keep-alive async connections, combined with high CPU. You may want to lower the maximum number of simultaneous connections to the server (MaxRequestWorkers), and/or decrease the KeepAliveTimeout to avoid holding connections open longer than necessary.\n\n",
+	"tags": [
+      "integration:apache"
+    ],
+	"options": {
+		"notify_audit": false,
+		"locked": false,
+		"timeout_h": 0,
+		"new_host_delay": 300,
+		"require_full_window": true,
+		"notify_no_data": false,
+		"renotify_interval": "0",
+		"escalation_message": "",
+		"no_data_timeframe": null,
+		"include_tags": true,
+		"thresholds": {
+			"critical": 1,
+			"warning": 0.8,
+			"critical_recovery": 0,
+			"warning_recovery": 0.6
+		},
+		"threshold_windows": {
+			"trigger_window": "last_15m",
+			"recovery_window": "last_15m"
+		}
+	},
+	"recommended_monitor_metadata": {
+        "description": "Notify your team when the number of idle workers is running low."
```

Comment:
Is this a copy/paste from the previous monitor? Maybe this should be "Notify your team when the number of keep-alive async connections and the CPU load are both running high."